### DISCUSSION TITLE
StatusBar config

### DISCUSSION THREAD

**Discussion by royalrex** [ionic-team/capacitor#3564] [Q&A] [answered]
Hi.

Is there any config parameters available for controlling the statusbar style?

I am hoping to set it in capacitor.config.json and have the plugin use it on startup. Is that possible?

Thank you.

**Comment by jcesarmobile** [comment, marked as answer]
You can configure it on the Info.plist
https://developer.apple.com/documentation/bundleresources/information_property_list/uistatusbarstyle

**Reply by royalrex** [reply]
okay, got it!

still wrapping my head around capacitor - coming from cordova.

Still would be nice to be able to set defaults in capacitor.config.json file :)

**Reply by jcesarmobile** [reply]
The capacitor.config.json is not read until the app starts, so that wouldn’t affect the statusbar style on the splash screen
